output/test.js

```diff
@@ -1,6 +1,6 @@
-var getIt = require('../');
+var index = require('../index.js');
```

Comment:
`index` isn't a great variable name (it doesn't tell you what the variable is!) I would use something like `fetcher`.